src/client/Client.cc

```diff
@@ -11441,7 +11441,6 @@ int Client::ll_link(Inode *in, Inode *newparent, const char *newname,
   if (r == 0) {
     assert(target);
     fill_stat(target, attr);
-    _ll_get(target.get());
```

Comment:
Can you pull out this patch for later discussion? You're probably right we have a problem here with non-fuse callers, but this fix doesn't look right either and isn't really in theme with the others.